.ci/jenkins_job_build.sh

```diff
@@ -112,7 +112,12 @@ fi
 # checks, as we always want to run those.
 # Work around the 'set -e' dying if the check fails by using a bash
 # '{ group command }' to encapsulate.
-{ .ci/ci-fast-return.sh; ret=$?; } || true
+if [ "${kata_repo}" == "${tests_repo}" ]; then
+	{ .ci/ci-fast-return.sh; ret=$?; } || true
+else
+	{ ${GOPATH}/src/${tests_repo}/.ci/ci-fast-return.sh; ret=$?; } || true
```

Comment:
This is correct, but I'd be tempted to reduce this down to a single code-path and always just call the following since that will also work for `tests`:

```sh
{ ${GOPATH}/src/${tests_repo}/.ci/ci-fast-return.sh; ret=$?; } || true
```